augur/tasks/start_tasks.py

```diff
@@ -210,6 +210,34 @@ def primary_repo_collect_phase(repo_git):
     #A chain is needed for each repo.
     repo_info_task = collect_repo_info.si(repo_git)#collection_task_wrapper(self)
 
+### Section from traffic metric merge that may need to be changed
```

Comment:
This is old and this new section needs to be removed.

Reply:
@IsaacMilarky : I merged from `dev` into this branch and I think that would fix this .